NBitcoin/Script.cs

```diff
@@ -117,6 +117,13 @@ public enum ScriptVerify : uint
 		/// Public keys in segregated witness scripts must be compressed
 		/// </summary>
 		WitnessPubkeyType = (1U << 15),
+		
+		/// <summary>
+		/// SCRIPT_VERIFY_FORKID flag for BPrivate
+		/// https://github.com/BTCPrivate/BitcoinPrivate/blob/4031ff02ec7c56bcafa085b01100cbddfcd33ea3/src/script/interpreter.h#L130
+		/// https://github.com/BTCGPU/BTCGPU/commit/777899b682c3d59702a4223c8239705a74616a63#diff-b81dfdd8a5bd80fe9f82b5a40c4c991eR126
+		/// </summary>
+		BTCPForkId = (1U << 16),

```

Comment:
Remove this, this is not used anywhere, and `Script.cs` should not be polluted by altcoin specific logic.

Reply:
Note that this is used as a magic number in https://github.com/MetacoSA/NBitcoin/blob/4b989108b44d7add50f28b777e0495d09949f311/NBitcoin/Policy/StandardTransactionPolicy.cs#L194

Reply:
(Done)